pfio/v2/fs.py

```diff
@@ -87,9 +87,16 @@ class FS(abc.ABC):
     '''
 
     _cwd = ''
+    _scheme = ''
 
-    def __init__(self):
+    def __init__(self, scheme=None):
         self.pid = os.getpid()
+        if scheme:
+            self._scheme = str(scheme)
+
```

Comment:
I'd like you to add a few sentences of document here to clarify this variable stores the name of custom scheme or actual implementation scheme (e.g. s3, data, or hdfs). Reading the code told me either can be here. The former if custom scheme defined, and the latter if it's not defined.